index.html

```diff
@@ -149,18 +149,18 @@ <h4>
 									More Softwares
 								</h4>
 								<p>
-									It's based on arch linux. so, with AUR you'll have a lots of amazing softwares you won't find on any other distro.
+									Being based on Arch Linux, you'll have lots of amazing software you won't find on other distributions.
```

Comment:
‘With ArchCraft being based…’?

Also, I’d use singular ‘software’ also in the header.

Reply:
Yeah I like this

Reply:
Problem with "With Archcraft being based" is that it goes over too many lines, so the rect height doesn't line up.  Will think about maybe making a compromise though

Reply:
I kept this as is in latest commit, but definitely can be changed if you find a better compromise.

Reply:
*Archcraft, being based on Arch Linux, offers lots of amazing software you won’t find on other distributions.*

Can be? If you need it even shorter, omit *being* and replace *offers* with *has*.

Reply:
It works with this